server/database/models/user.js

```diff
@@ -9,7 +9,23 @@ const userSchema = new Schema({
   password: { type: String, unique: false, required: false },
   name: { type: String, unique: false, required: false },
   email: { type: String, unique: false, required: false },
-});
+  publications: {
+    id: { type: Number, unique: true, required: true },
+    name: { type: String, unique: false, required: false },
+    twitterHandle: { type: String, unique: true, required: true }
+  },
```

Comment:
[discussion] Add this to our next Zoom list of discussion topics :smile:. The `User` model is doing _too much_ if it's defining the model structure for publications, articles, etc. Instead, we should be linking a User object to Publication/Article/Topic objects. I'll have to research how this is done in Mongo because I'm not sure off hand. The result is a User document simply maintains a list of IDs for their publications, articles and topics, that behave like "foreign keys". A lot to unpack here.

Reply:
the `user` model will bring in foreign keys from the Publication/Article/Topic models, but I hard-coded them into `user` model until the others are written. Will clean up when I move to the next step.